modules/ROOT/pages/dw-core-functions-orderby.adoc

```diff
@@ -40,38 +40,28 @@ array. Note that `orderBy($.letter)` produces the same result as
 %dw 2.0
 output application/json
 ---
-{ letter: "b", letter: "c", letter: "a" } orderBy ((value, key) -> value)
+[{ letter: "e" }, { letter: "d" }] orderBy($.letter)
 ----
 
 ==== Output
 
 [source,JSON,linenums]
 ----
-{
-  "letter": "a",
-  "letter": "b",
-  "letter": "c"
-}
+[
+  {
+    "letter": "d"
+  },
+  {
+    "letter": "e"
+  }
+]
 ----
 
-
```

Comment:
pls see if wee should restore this or if it was removed from source

Reply:
This looks present in the sources. Let's keep it and have @mlischetti take a look.

Reply:
Sorry, it looks like the order changed and now it's below in line 83